benchmarks/quant-hela-method-comparison/scripts/06_method_correlation_plots.py

```
@@ -0,0 +1,524 @@
+    filename = f"correlation_all_{method1}_vs_{method2}"
+    plt.savefig(output_dir / f"{filename}.png", dpi=150, bbox_inches="tight")
+    plt.savefig(output_dir / f"{filename}.pdf", bbox_inches="tight")
+    plt.close()
+
+    print(f"  Saved: {filename}.png/pdf")
+
+    return pd.DataFrame(results)
+
+
+def create_method_comparison_grid(
+    methods: List[str] = None,
```

Ruff (0.14.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)